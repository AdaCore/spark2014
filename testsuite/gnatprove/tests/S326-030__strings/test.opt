ALL XFAIL test not implemented yet
