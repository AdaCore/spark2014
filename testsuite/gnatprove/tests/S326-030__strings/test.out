test_string.ads:8:13: info: implicit aspect Always_Terminates on "Get_Result_Text" has been proved, subprogram will terminate
